packages/salesforcedx-vscode-apex/src/commands/apexActionController.ts

```diff
@@ -274,4 +298,38 @@ export class ApexActionController {
     const builder = new XMLBuilder({ ignoreAttributes: false, format: true });
     return builder.build(jsonObj);
   };
+
+  private getOrgVersion = async (): Promise<string | undefined> => {
```

Comment:
This is a lot of code for something that should be available already. Could you look into other means of getting this. Maybe through a core api?

Reply:
getAuthFieldsFor() is in core package but I was getting errors using that one. I also see it's already duplicated (orgList.ts and orgUtil.ts). Let me see what else I can do